### PR TITLE
Users can check off all completables (resource, skills, etc).

### LINKED ISSUE #163: Allow users to check off resources once they've been consumed
Copied from Nova PR:

Finally get around to putting checkboxes on all of the resources/etc. now that the data is purely represented in the database, not in seeders (and therefore has reliable and consistent foreign key IDs)

### PR DESCRIPTION
**Pull request by TammyTee**
Once merged, this resolves #163.

### CODE REVIEW

File: resources/views/modules/show.blade.php [hunk 1]
```diff
@@ -27,16 +27,24 @@
             </resource-language-preference-switcher>
             <div class="flex">
                 <div class="flex-1 w-auto p-4 border rounded mr-2">
-                    <h3 class="font-bold text-lg border-b mb">
+                    <h3 class="font-bold text-lg border-b mb-3">
                         Skills
                     </h3>
                     <ul>
```
Comment (mattstauffer) [inline resources/views/modules/show.blade.php]:
```suggestion
                    <ul class="@guest list-disc @endguest">
```

File: resources/views/modules/show.blade.php [hunk 2]
```diff
@@ -27,16 +27,24 @@
...
-                                <!--input type="checkbox"{{ $completedSkills->contains($skill->id) ? ' checked="checked"' : '' }}-->
+                                <completed-checkbox
+                                        :initial-is-completed="{{ $completedSkills->contains($skill->id) ? 'true' : 'false' }}"
+                                        type="{{ $skill->getMorphClass() }}"
+                                        id="{{ $skill->id }}"
+                                        ></completed-checkbox>
                                 @endauth
-                                &bull; {{ $skill->name }}
+
+                                @guest
```
Comment (mattstauffer) [inline resources/views/modules/show.blade.php]: Delete these lines as a result of using list-disc

File: resources/views/partials/resource-on-module-page.blade.php [hunk 3]
```diff
@@ -6,5 +6,9 @@
         id="{{ $resource->id }}"
         ></completed-checkbox>
     @endauth
+
+    @guest
```
Comment (mattstauffer) [inline resources/views/partials/resource-on-module-page.blade.php]: Same here, please move to list-disc

**Comment by mattstauffer** [comment]
Great work!!

**Comment by TammyTee** [comment]
@mattstauffer all set here. merging into your feature branch.

**Pull request merged**